cirq-core/cirq/vis/heatmap.py

```diff
@@ -36,6 +38,9 @@
 from cirq.devices import grid_qubit
 from cirq.vis import vis_utils
 
+if TYPE_CHECKING:
+    from numpy.typing import ArrayLike
```

Comment:
Can't do this in an `if TYPE_CHECKING` block unless you also make annotations lazy by doing `from __future__ import annotations`.

Reply:
Done, thank you.